functions/Get-DbaServerInstallDate.ps1

```diff
@@ -0,0 +1,181 @@
+function Get-DbaServerInstallDate
+{
+<#
+.SYNOPSIS
+Returns the install date of SqlServer and Windows Server, depending on what is passed. 
+	
+.DESCRIPTION
+By default, this command returns for each SQL Server instance passed in:
+SQL Instance install date, formatted as a string
+Hosting Windows server install date, formatted as a string
+	
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER SqlCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER WindowsCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER SqlOnly
```

Comment:
You could shorten it by saying that default most are going to want to know when SQL Server was installed. So that is part of your default output, then just have an `-IncludeWindows` to show the install date for that part.